### PR TITLE
Import strings from android-l10n

### PR DESCRIPTION
**Pull request by Pike**
State: mozilla-l10n/android-l10n@d10506f5697e7c2a286d4934755404bd768246fa

### CODE REVIEW

File: app/src/main/res/values-da/strings.xml [hunk 1]
```diff
@@ -636,11 +628,24 @@
     <!-- This string is displayed as the title of the Bookmarks List, which contains the user's bookmarked websites. -->
     <string name="bookmarks_title">Bogmærker</string>
 
+    <!-- This string is displayed as the title of the Bookmarks List when the bookmarks list is empty. -->
+    <string name="bookmarks_empty">Din liste med bogmærker er tom</string>
+
+    <!-- This string is displayed under the bookmarks title in the bookmarks list. -->
+    <string name="bookmarks_description">Klik på @-ikonet for at tilføje et bogmærke til den side, du er på.</string>
```
Comment (keianhzo) [inline app/src/main/res/values-da/strings.xml]: Is the "-" specific to some languages? I see that most of them don't use it but some do.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -885,4 +978,18 @@
 
     <!-- This string is displayed in the content language list in front of the system default language in between parenthesis -->
     <string name="language_default">Standard</string>
+
+    <!-- This string is displayed in the URL bar when the user is in the home page.
+         '%1$s' will be replaced at runtime with the app's name. -->
+    <string name="url_home_title">%1$s-Startseite</string>
```
Comment (keianhzo) [inline app/src/main/res/values-de/strings.xml]: Is the "-" specific to some languages? I see that most of them don't use it but some do.

File: app/src/main/res/values-nb-rNO/strings.xml [hunk 3]
```diff
@@ -638,11 +638,24 @@
     <!-- This string is displayed as the title of the Bookmarks List, which contains the user's bookmarked websites. -->
     <string name="bookmarks_title">Bokmerker</string>
 
+    <!-- This string is displayed as the title of the Bookmarks List when the bookmarks list is empty. -->
+    <string name="bookmarks_empty">Bokmerkelisten din er tom</string>
+
+    <!-- This string is displayed under the bookmarks title in the bookmarks list. -->
+    <string name="bookmarks_description">Klikk på @-ikonet når du er på en side for å legge det til i bokmerker.</string>
```
Comment (keianhzo) [inline app/src/main/res/values-nb-rNO/strings.xml]: Is the "-" specific to some languages? I see that most of them don't use it but some do.

File: app/src/main/res/values-nb-rNO/strings.xml [hunk 4]
```diff
@@ -888,4 +981,18 @@
 
     <!-- This string is displayed in the content language list in front of the system default language in between parenthesis -->
     <string name="language_default">Standard</string>
+
+    <!-- This string is displayed in the URL bar when the user is in the home page.
+         '%1$s' will be replaced at runtime with the app's name. -->
+    <string name="url_home_title">%1$s-startside</string>
```
Comment (keianhzo) [inline app/src/main/res/values-nb-rNO/strings.xml]: Is the "-" specific to some languages? I see that most of them don't use it but some do.

File: app/src/main/res/values-sv-rSE/strings.xml [hunk 5]
```diff
@@ -636,11 +636,24 @@
     <!-- This string is displayed as the title of the Bookmarks List, which contains the user's bookmarked websites. -->
     <string name="bookmarks_title">Bokmärken</string>
 
+    <!-- This string is displayed as the title of the Bookmarks List when the bookmarks list is empty. -->
+    <string name="bookmarks_empty">Din bokmärkeslista är tom</string>
+
+    <!-- This string is displayed under the bookmarks title in the bookmarks list. -->
+    <string name="bookmarks_description">Klicka på @-ikonen när du vill lägga till den i dina bokmärken.</string>
```
Comment (keianhzo) [inline app/src/main/res/values-sv-rSE/strings.xml]: Is the "-" specific to some languages? I see that most of them don't use it but some do.

**Comment by Pike** [comment]
Yes, hyphenation rules are language specific

**Pull request merged**